easybuild/easyconfigs/a/ADIOS/ADIOS-20210804-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,73 @@
+# Based on ADIOS-1.13.1-foss-2019a-Python-2.7.15.eb
+# Updated to foss-2020a toolchain and switched to ConfigureMake
+# Latest version from GitHub
+# Mini-XML updated to latest 2.x. version
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'ADIOS'
+version = '20210804'
+local_commit = 'de85222'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.olcf.ornl.gov/center-projects/adios/'
+description = """The Adaptable IO System (ADIOS) provides a simple,
+flexible way for scientists to describe the data in their code that may
+need to be written, read, or processed outside of the running
+simulation."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+github_account = 'ornladios'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+
+patches = [
+    '%(name)s-1.13.1_force_use_of_mpi.patch',
+    '%(name)s-1.13.1_fix_missing_thread_lib.patch',
+    '%(name)s-1.13.1_fix_search_for_szip.patch',
+]
+checksums = [
+    'c58af40cbdb13843de2c2c157379d069880835c9b932ad05e774910ec23a468f',  # v1.13.1.tar.gz
```

Comment:
@sassy-crick The filename in the comment is definitely wrong here...
Easiest way to correct this is using `--force --inject-checksums`

Reply:
@boegel Thanks for spotting that. I am trying to avoid the `--inject-checksums` as that way I prevent getting things wrong. I usually download the tarball first, look at it, do the checksum and update the EC file. At least when I get things wrong, the checksum will tell me. Different way of working I guess. 
I did not know that the command line option also sets the comment though. So thanks for that bit of information.